docs/layers/column-layer.md

```diff
@@ -133,6 +140,16 @@ to scale all hexagon elevations without updating the data.
 
 Whether to extrude hexagon. If se to false, all hexagons will be set to flat.
 
+##### `wireframe` (Boolean, optional)
+
+* Default: `false`
+
+Whether to generate a line wireframe of the column. The outline will have
+"horizontal" lines closing the top and bottom polygons and a vertical line
+(a "strut") for each vertex on the polygon.
+
+Requires the `extruded` prop to be true.
```

Comment:
It is supposed to be working with non extruded mode?

Reply:
Not until the stroke is being implemented. I am doing a follow-up PR for the update in H3Hexagon docs, will edit this as well in the PR.